db/postgres_schema.rb

```diff
@@ -747,6 +747,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.bigint "tenant_id"
+    t.index ["account_id"], name: "index_mail_dispatch_rules_on_account_id"
     t.index ["system_operation_id", "account_id"], name: "index_mail_dispatch_rules_on_system_operation_id_and_account_id", unique: true
```

Comment:
What if we index it by `["account_id", "system_operation_id"]`? In this way one index will cover both - searching by account_id only and by both.

Does anything query by `system_operation_id` but not `account_id`?

Reply:
Yes, I thought about this, but thought just adding a new one was easier.
(i.e. I was lazy to check whether there is any usage where `system_operation_id` was being checked only, the initial quick search didn't reveal any)

Reply:
I can change that. I think even with `["account_id", "system_operation_id"]` index, the search will not be as efficient by just `account_id` as with a "dedicated" index, but should be good enough, as `system_operation_id` has quite a small number of options (I think it has just 16 possible values)

Reply:
It should be relatively as efficient as a single index. There is a little more IO in a composite index because a concatenated index is longer. It should be negligible though.